spec/lucky_web/number_to_currency_spec.cr

```diff
@@ -0,0 +1,62 @@
+require "../../spec_helper"
+
+private class TestPage
+  include LuckyWeb::HTMLPage
+
+  def render
+  end
+end
+
+describe LuckyWeb::NumberToCurrency do
+  describe "number_to_currency" do
+    it "accepts Float" do
+      view.number_to_currency(29.92).should eq "$29.92"
+    end
+
+    it "accepts String" do
+      view.number_to_currency("92.29").should eq "$92.29"
+    end
+
+    it "accepts Integer" do
+      view.number_to_currency(92).should eq "$92.00"
+    end
+
+    it "accepts Nil" do
```

Comment:
Should this be allowed?

Reply:
Good question. I think it should not be allowed